Formula/mpd.rb

```diff
@@ -25,6 +25,7 @@ class Mpd < Formula
   option "with-yajl", "Build with yajl support (for playing from soundcloud)"
   option "with-opus", "Build with opus support (for Opus encoding and decoding)"
   option "with-libmodplug", "Build with modplug support (for decoding modules supported by MODPlug)"
+  option "with-pulseaudio", "Build with modplug support (for decoding modules supported by MODPlug)"
```

Comment:
This seems to need a description update, like "Build with PulseAudio support (for sending audio output to a PulseAudio sound server)".